requirements/pip_tools.in

```diff
@@ -1,3 +1,3 @@
 # Dependencies to run compile tools
-pip-tools<5.0.0                           # Contains pip-compile, used to generate pip requirements files
+pip-tools<=5.2.1                          # Contains pip-compile, used to generate pip requirements files
```

Comment:
A nitpick about this, but I think we should just remove the constraint on this requirement if it's not necessary any longer.

Reply:
Done, current constraint was removed.